package.json

```diff
@@ -29,15 +29,20 @@
   "homepage": "https://github.com/hoodiehq/hoodie-server-store#readme",
   "devDependencies": {
     "coveralls": "^2.11.6",
+    "memdown": "^1.1.2",
     "nock": "^7.0.1",
     "nyc": "^5.3.0",
+    "request": "^2.69.0",
     "semantic-release": "^6.0.3",
     "standard": "^6.0.4",
     "tap": "^5.0.1"
   },
   "dependencies": {
+    "boom": "^3.1.2",
+    "express": "^4.13.4",
+    "express-pouchdb": "^1.0.2",
     "h2o2": "^5.0.0",
     "hapi": "^13.0.0",
-    "joi": "^8.0.1"
+    "pouchdb": "^5.2.1"
```

Comment:
yeah, you are right, this is a much more convenient way. thank you for the feedback. :ok_hand: